.github/workflows/IOnIntegration.yml

```diff
@@ -0,0 +1,94 @@
+name: I-On Integration
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+    tags:
+      - v*
+
+env:
+  IMAGE_NAME: i-on-integration
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Docker image
+        id: build_image
+        run: docker build . --file Dockerfile
+
+  push_integration:
+    needs: build
+
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'heads')
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build image
+        id: integration_build_image
+        run: docker build . --file Dockerfile --tag image
```

Comment:
I think the default behavior of docker build is to look for a file named Dockerfile in the context. So adding --file Dockerfile is redundant

Reply:
I'll remove it